cmd/juju/application/deployer/register.go

```diff
@@ -81,27 +80,6 @@ func (r *RegisterMeteredCharm) RunPre(api DeployStepAPI, bakeryClient *httpbaker
 		if r.Plan == "" && info.Metrics != nil && !info.Metrics.PlanRequired() {
 			return nil
 		}
-
```

Comment:
This entire file can be removed. It's part of the Omnibus work. https://chat.charmhub.io/charmhub/pl/at8595ckyjbbm8ut98o51f5mca

Is it easier to do now or in a follow on pr?

Reply:
https://github.com/juju/juju/pull/15013/commits/d8ee6930d7c0aad59bfb7e625b18365de9425362 clears it out :+1: